SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -1324,12 +1328,14 @@ - (void)saveGroupMessage:(TSOutgoingMessage *)message inThread:(TSThread *)threa
         // TODO: Why is this necessary?
         [message save];
     } else if (message.groupMetaMessage == TSGroupMessageQuit) {
-        [[[TSInfoMessage alloc] initWithTimestamp:message.timestamp
+        // We want the info message to appear after the message.
```

Comment:
I don't think these two offsets are necessary, since, if it's not TSGroupMessageDeliver, there is no corresponding visible "message" to come "after". 

No immediate problems with offsetting these timestamps by 1ms. The only thing I could think of is it might make debugging trickier down the road when trying to cross reference something stored in the DB with a log. No biggie if you'd rather keep them like this for some reason.

Reply:
Done.